salt/modules/mac_brew_pkg.py

```diff
@@ -138,6 +147,55 @@ def _list_pkgs_from_context(versions_as_list):
         return ret
 
 
+def homebrew_prefix(raise_err=True):
+    """
+    Returns the full path to the homebrew prefix
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' pkg.homebrew_prefix
+    """
+
+    # Try HOMEBREW_PREFIX env variable
+    env_homebrew_prefix = "HOMEBREW_PREFIX"
+    if env_homebrew_prefix in os.environ:
+        log.debug(f"{env_homebrew_prefix} is set. Using it for homebrew prefix.")
+        return os.environ[env_homebrew_prefix]
+    else:
+        log.warn(f"{env_homebrew_prefix} is not set. Please, consider adding it.")
+
+    # Try brew --prefix
```

Comment:
Homebrew is not intended to be run as root, so I prefer to check for the environment variable before running `brew` as root.